kzg4844.nimble

```diff
@@ -0,0 +1,45 @@
+mode = ScriptMode.Verbose
+
+##################################################
+# Package definition
+##################################################
+
+packageName   = "kzg4844"
+version       = "0.1.0"
+author        = "Andri Lim"
+description   = "Nim wrapper of c-kzg-4844"
+license       = "Apache License 2.0"
+skipDirs      = @[
+  "tests", "lib", "inc", "fuzz",
+  "bindings/csharp",
+  "bindings/go",
+  "bindings/java",
+  "bindings/node.js",
+  "bindings/python",
+  "bindings/rust"
+  ]
+installDirs   = @[
+  "blst",
```

Comment:
With nimble, will this directory (submodule) be initialized? For Go, I had to copy the blst headers to here:

* https://github.com/ethereum/c-kzg-4844/tree/main/bindings/go/blst_headers

Reply:
Yes, nimble can handle submodule directory. I've tested this package configuration, and it's works.
But other nimbus team member prefer that the `.nimble` file be put into `bindings/nim`, and those ugly proxy modules with import and reexport can be removed.
But sadly, putting `.nimble` file into `bindings/nim` doesn't work for the moment, because nimble can't install `blst` and `src` from `bindings/nim`.
So we have to ask nimble team to take a look at this issue.

Reply:
That's really cool that nimble can handle that. And ah, I understand. Sounds good 👍